app/views/posts/index.html.erb

```diff
@@ -1,3 +1,4 @@
+<% @posts.each do |post| %>
 <% @post = post %>
```

Comment:
Is this necessary? Can't we just use the local post variable for the second nested #each?